build-parent/pom.xml

```diff
@@ -788,6 +788,15 @@
                     <groupId>org.jboss.jandex</groupId>
                     <artifactId>jandex-maven-plugin</artifactId>
                     <version>1.0.5</version>
+                    <dependencies>
+                        <!-- For now we need to override and hardcode the jandex version -->
+                        <!-- See also https://github.com/wildfly/jandex-maven-plugin/pull/13 -->
+                        <dependency>
+                          <groupId>org.jboss</groupId>
+                          <artifactId>jandex</artifactId>
+                          <version>2.1.1.Final</version>
```

Comment:
Minor but this is not properly indented: it should be 4 spaces. Could you fix it?

I think we need to fix having the version defined here but let's discuss that later and get this in.

Reply:
shouldn't it use `${jandex.version}` ?

Reply:
Ah ah, I just asked about it in a previous review. We can't as it's defined in the bom and properties from the bom are not inherited.

I'm thinking about moving the version properties to the quarkus-parent artifact but that's an orthogonal discussion and we need this fixed sooner rather than later.

Reply:
ah right.
Why is the plugin not releasing in alignment of Jandex? We should ask @n1hility : Would be nicer to have the plugin version share the version of jandex itself, so to guarantee synch (and express compatibility)

Reply:
Yeah, it's a separate project for now but I could see how it would be nice to have it as one project.

We have a PR waiting here: https://github.com/wildfly/jandex-maven-plugin/pull/13#issuecomment-480904885 .

Reply:
@gsmet Fixed :-)